package/kernel/linux/modules/netsupport.mk

```diff
@@ -885,7 +885,7 @@ $(eval $(call KernelPackage,bpf-test))
 define KernelPackage/sched
   SUBMENU:=$(NETWORK_SUPPORT_MENU)
   TITLE:=Extra traffic schedulers
-  DEPENDS:=+kmod-sched-core +kmod-ipt-core +kmod-lib-crc32c
+  DEPENDS:=+kmod-sched-core +kmod-ipt-core +kmod-lib-crc32c +tc
```

Comment:
This is just a package dependency choice, not a question of kernel/userspace layering or similar. 😃 

The key change is that now the `teql` module and `hotplug` script are bundled together in `kmod-sched`. It's almost certain that a system installing `kmod-sched` already has `tc` installed, so while it could be omitted, adding "+tc" just makes it certain and explicit.

The previous setup had the `hotplug` script pushed into `tc`, and it was very possible for someone to install `tc` without installing `kmod-sched` and the `teql` module. That's clearly a worse situation.

Both @jow- and @ldir-EDB0 were part of the prior discussion in #1627 , so I'll take their advice on whether the "+tc" dependency is wanted.

Reply:
Another thought it to create a package that depends on the kmod and tc and that installs the script.

Reply:
But that would split the components (script, module, tc) up into 3 different packages, which is arguably worse than what's proposed, given the current `kmod-sched` is also a monolithic, kitchen-sink package that would install lots of unneeded modules if selected by `teql-scripts`. Honestly, I would be fine with just dropping the "+tc" for now since it's unlikely to be an issue. Would that work?

Longer term, I wanted to break apart giant packages like `kmod-sched`, and e.g pull out a small `kmod-sched-teql`, which would fit nicely with your dependency idea: `tc` <-- `teql-scripts` --> `kmod-sched-teql`. But I don't think there's time enough to do all that *and* backport `iproute2` for 21.02 as I'd like.

Reply:
After considering this for a while, I've chosen the simplest option and dropped the tc dependency to move things forward.